torch/ao/quantization/quantizer/xnnpack_quantizer.py

```diff
@@ -341,6 +342,11 @@ def set_module_name(
         self.module_name_config[module_name] = quantization_config
         return self
 
+    def transform_for_annotation(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
+        """Transforms scalar values to tensor attributes
```

Comment:
can you also add a test to xnnpack quantizer?

Reply:
yeah, I have this in my local PR, will update

Reply:
for the test we should also include a test where the model performs add/mul against constant sys.float_max

Reply:
@mcr229 is that a different issue? I think we can have a separate PR for that

Reply:
sure but i think we should add a test for that pass, with edge cases like float.sysmax/min constants